integration-tests/elasticsearch-rest-client/src/test/java/io/quarkus/it/elasticsearch/FruitResourceTest.java

```diff
@@ -56,6 +56,20 @@ public void testEndpoint() throws InterruptedException {
         Assertions.assertEquals("1", results.get(0).id);
         Assertions.assertEquals("Apple", results.get(0).name);
         Assertions.assertEquals("Green", results.get(0).color);
+
+        given()
+                .contentType("application/json")
+                .body(List.of(fruit))
+                .when().post("/fruits/bulk")
+                .then()
+                .statusCode(200);
+
+        given()
+                .contentType("application/json")
+                .body(List.of(fruit.id))
+                .when().delete("/fruits/bulk")
+                .then()
+                .statusCode(200);
```

Comment:
I probably wouldn't suggest this one, but since we'd have to update the ndjson examples in the doc file ...
Maybe we can create a new fruit index / check it exists / delete / check it doesn't exist anymore ?

see how just above `Fruit result = get("/fruits/1").as(Fruit.class);` is used to get the fruit (we can use that to check if it exists)
you'll need to wait for the index to refresh so you may want to use awaitlity for that instatead of the `Thread.sleep()`, e.g.:
```java
await().atMost(5, TimeUnit.SECONDS).untilAsserted(() -> {
            get("/fruits/1").statusCode(200);
        });
```

Reply:
wow, that's nicee